File: mainherencia.py
from Herencia import*

def main():
    print("costo Llamada Telefonica")
    costo=LlamadaTelefonica(20,3)
    print(costo.getCostoLLamadaTelefonica())

    print("Costo Telefono Fijo")
    costo2=TelefonoFijo(20,2)
    print(costo2.getCostoTelefonoFijo())

if __name__== "__main__":
    main()

File: mainHERENCIA.java
package HERENCIA;


public class mainHERENCIA {
    public static void  main(String[]args) {
        System.out.println("costollamada");
        LlamadaTelefonica resultado = new LlamadaTelefonica(34,2);
        System.out.println(resultado.getCostoLlamada());

        System.out.println("TelefonoFijo");
        TelefonoFijo resultado2= new TelefonoFijo(35,2);
        System.out.println((resultado2.getTelefonoFijo()));
    }
}
